Tools/LyTestTools/ly_test_tools/o3de/editor_test.py

```diff
@@ -876,12 +878,27 @@ def _exec_editor_multitest(self, request: _pytest.fixtures.FixtureRequest,
         editor_utils.cycle_crash_report(run_id, workspace)
 
         results = {}
-        test_filenames_str = ";".join(editor_utils.get_testcase_module_filepath(test_spec.test_module) for test_spec in test_spec_list)
+        temp_batched_file = None
+
+        # The Editor cannot handle more than 10 args, so we instead create a file containing a semicolon separated list
+        if len(test_spec_list) > 10:
```

Comment:
+1.... this isn't a bad idea, the only problem I can see is would this require all existing tests to be updated to use this approach? (I'm not sure how many there are but this could potentially be prohibitive).

It might otherwise we could to 'reduce' the vector or `test_spec_list` to a string that contains all the names to test the size/number of characters? That may be more robust and solve the issue moraaar mentioned.

Reply:
The only update to tests would be to switch them to use the EditorBatch mode. I'llchange it to always using a file as mentioned above for consistency and simplification.